scripts/aws/cloudformation/tasking-manager.template.js

```diff
@@ -340,6 +340,7 @@ const Resources = {
       UserData: cf.userData([
         '#!/bin/bash',
         'set -x',
+        'sleep 60',
```

Comment:
We could optionally complicate things by writing a check to see if apt process is locked. Something like this: https://gist.github.com/tedivm/e11ebfdc25dc1d7935a3d5640a1f1c90